dbcon/mysql/ha_mcs_client_udfs.cpp

```diff
@@ -924,6 +924,103 @@ extern "C"
   {
   }
 
+  // --- Plan UDFs ---
+  // Return plan strings or applied rules based on optional parameter:
+  //   no arg or 1 -> optimized/latest plan (ci->queryPlan)
+  //   0 or 'original'  -> pre-RBO plan
+  //   1 or 'optimized' -> post-RBO plan
+  //   2 or 'rules'     -> comma-separated applied RBO rules
+  const char* mcsgetplan(UDF_INIT* /*initid*/, UDF_ARGS* args, char* /*result*/, unsigned long* length,
```

Comment:
```suggestion
  const char* mcs_get_plan(UDF_INIT* /*initid*/, UDF_ARGS* args, char* /*result*/, unsigned long* length,
```

Reply:
You are welcome to change it with all 
the others like   const char* mcsgetstats(UDF_INIT* /*initid*/, UDF_ARGS* /*args*/, char* result, unsigned long* length,
In the separate PR

Reply:
Feel free to check out naming style for all new UDFs recently added.
[Here](https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/stable-23.10/dbcon/mysql/ha_mcs_client_udfs.cpp#L559) is the example.

Reply:
I save the style of the file I deal with. You can add your naming convention on your own